x/tss/client/cli/query.go

```diff
@@ -407,3 +408,35 @@ func GetCmdNextKeyID(queryRoute string) *cobra.Command {
 	flags.AddQueryFlagsToCmd(cmd)
 	return cmd
 }
+
+// GetCmdGovernanceKey returns the governance key of the network
+func GetCmdGovernanceKey(queryRoute string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "governance-key",
+		Short: "Returns the governance key",
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			path := fmt.Sprintf("custom/%s/%s", queryRoute, keeper.QueryGovernanceKey)
+
+			bz, _, err := clientCtx.Query(path)
+			if err != nil {
+				return sdkerrors.Wrap(err, "could not get governance key")
+			}
+
+			var res types.QueryGovernanceKeyResponse
+			if err := res.Unmarshal(bz); err != nil {
+				return sdkerrors.Wrapf(err, "failed to get governance key")
+			}
+
+			return clientCtx.PrintProto(&res)
+		},
+	}
+
+	flags.AddQueryFlagsToCmd(cmd)
+	return cmd
+}
```

Comment:
could we please not implement new legacy queries anymore? Ask @jcs47 for help to convert to grpc query 🙂